package.json

```diff
@@ -3,107 +3,90 @@
+    "@types/react-dom": "^16.9.8",
+    "@types/react-intl": "^2.3.18",
+    "@types/react-leaflet": "^2.5.1",
+    "@types/react-redux": "^7.1.9",
     "@types/react-router": "^4.4.5",
-    "@types/react-router-dom": "^4.3.1",
-    "@types/react-test-renderer": "^16.8.1",
-    "@types/react-transition-group": "^2.0.16",
-    "@types/reactstrap": "^7.1.3",
-    "@types/recompose": "^0.30.5",
-    "@types/redux": "^3.6.0",
-    "@types/redux-actions": "^2.6.0",
-    "@types/redux-persist": "^4.3.1",
-    "@types/redux-promise-middleware": "^0.0.11",
-    "@types/redux-thunk": "^2.1.0",
-    "@types/typescript": "^2.0.0",
-    "apollo-boost": "^0.3.1",
+    "@types/react-router-dom": "^4.3.5",
+    "@types/react-test-renderer": "^16.9.2",
+    "@types/react-transition-group": "^4.4.0",
+    "@types/reactstrap": "^8.4.2",
+    "@types/recompose": "^0.30.7",
+    "@types/redux-actions": "^2.6.1",
+    "apollo-boost": "^0.4.9",
     "classnames": "^2.2.6",
-    "enzyme-adapter-react-16": "^1.11.2",
-    "final-form": "^4.18.2",
-    "graphql": "^14.2.1",
-    "history": "^4.7.2",
+    "enzyme-adapter-react-16": "^1.15.2",
+    "final-form": "^4.20.0",
+    "focus-visible": "5.1.0",
+    "graphql": "^15.1.0",
+    "history": "^4.10.1",
     "immutable": "^4.0.0-rc.12",
-    "leaflet": "^1.3.4",
-    "lodash": "^4.17.11",
-    "node-sass": "^4.10.0",
+    "leaflet": "^1.6.0",
+    "lodash": "^4.17.15",
+    "node-sass": "^4.14.1",
     "open-city-design": "^1.0.0-alpha.3",
     "piwik-react-router": "^0.12.1",
-    "react": "^16.8.6",
-    "react-apollo": "^2.5.3",
-    "react-app-polyfill": "^0.2.0",
-    "react-dom": "^16.8.6",
-    "react-final-form": "^6.3.0",
-    "react-intl": "^2.7.2",
-    "react-leaflet": "^2.1.2",
-    "react-redux": "^6.0.0",
+    "react": "^16.13.1",
+    "react-apollo": "^3.1.5",
+    "react-app-polyfill": "^1.0.6",
+    "react-dom": "^16.13.1",
+    "react-final-form": "^6.5.0",
+    "react-intl": "^2.9.0",
+    "react-leaflet": "^2.7.0",
+    "react-redux": "^6.0.1",
     "react-router": "^4.3.1",
     "react-router-dom": "^4.3.1",
-    "react-scripts": "2.1.8",
-    "react-test-renderer": "^16.8.6",
-    "react-transition-group": "^2.5.2",
-    "reactstrap": "6.5.0",
+    "react-scripts": "3.4.1",
+    "react-test-renderer": "^16.13.1",
+    "react-transition-group": "^4.4.1",
+    "reactstrap": "8.4.1",
     "recompose": "^0.30.0",
-    "redux": "^4.0.1",
-    "redux-actions": "^2.6.4",
+    "redux": "^4.0.5",
+    "redux-actions": "^2.6.5",
     "redux-persist": "^5.10.0",
     "redux-promise-middleware": "^5.1.1",
     "redux-thunk": "^2.3.0",
-    "typescript": "^3.4.1",
-    "focus-visible": "5.1.0"
+    "typescript": "^3.9.5"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "eject": "react-scripts eject",
     "prettier": "prettier",
     "lint": "tslint 'src/**/*.ts?(x)'",
-    "storybook": "start-storybook -p 9001 -c .storybook",
```

Comment:
There seems to be still references to storybook in README.md and in i18n file (key: `storybook.dummy.label`)